backend/czi_hosted/common/config/dataset_config.py

```diff
@@ -194,10 +199,7 @@ def handle_diffexp(self, context):
 
         server_config = self.app_config.server_config
         if server_config.single_dataset__datapath:
-            with server_config.matrix_data_cache_manager.data_adaptor(
-                self.tag, server_config.single_dataset__datapath, self.app_config
-            ) as data_adaptor:
-                if self.diffexp__enable and data_adaptor.parameters.get("diffexp_may_be_slow", False):
```

Comment:
This should be safe to remove because we arent launching any single dataset environments. The code on line 201  (previously 196) `if server_config.single_dataset__datapath:`  and the code below it will be removed when we do the hard split.

Reply:
In general, these heuristic warnings are not useful in the server environment (they just go into the logs or /dev/null)